esignet-default.properties

```diff
@@ -195,24 +197,22 @@ mosip.esignet.cache.store.individual-id=true
 mosip.esignet.cache.security.secretkey.reference-id=TRANSACTION_CACHE
 mosip.esignet.cache.security.algorithm-name=AES/ECB/PKCS5Padding
 
-mosip.esignet.ida.vci-user-info-cache=userinfo
-
 mosip.esignet.cache.names=clientdetails,preauth,authenticated,authcodegenerated,userinfo,linkcodegenerated,linked,linkedcode,linkedauth,consented,authtokens,bindingtransaction,vcissuance
 
-spring.cache.type=redis
-spring.cache.cache-names=${mosip.esignet.cache.names}
-spring.redis.host=redis-master-0.redis-headless.redis.svc.cluster.local
-spring.redis.port=6379
+#spring.cache.type=redis
+#spring.cache.cache-names=${mosip.esignet.cache.names}
+#spring.redis.host=localhost
```

Comment:
@Rakshitha650 let's enable REDIS, I suggest that all eSignet QA env should have REDIS connected so that we will not miss on the bugs when using Redis cache.